e2e/test/log.go

```diff
@@ -0,0 +1,40 @@
+package test
+
+import (
+	"os"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+const logFilePath = "ramen-e2e.log"
+
+func CreateLogger() (*zap.SugaredLogger, error) {
+	// Console encoder config
+	consoleConfig := zap.NewProductionEncoderConfig()
+	consoleConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	consoleConfig.EncodeLevel = zapcore.CapitalLevelEncoder
+	consoleConfig.CallerKey = zapcore.OmitKey
+	consoleEncoder := zapcore.NewConsoleEncoder(consoleConfig)
+
+	// Logfile encoder config
+	logfileConfig := zap.NewProductionEncoderConfig()
+	logfileConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+	logfileConfig.EncodeLevel = zapcore.CapitalLevelEncoder
+	logfileEncoder := zapcore.NewConsoleEncoder(logfileConfig)
+
+	// Create the log file in the current directory
```

Comment:
We can remove the comment, the code is self explaining, and the comment is wrong. This does not create the log file in the current directory, but at logFilePath. Because the current value is a relative path, it will be in the current directory, but once we change it, it may not be there.